tests/test_frame.py

```diff
@@ -1915,3 +1915,9 @@ def f(s: pd.DataFrame) -> pd.Series:
         return s.mean()
 
     check(assert_type(resampler.apply(f), Union[pd.Series, pd.DataFrame]), pd.DataFrame)
+
+
+def df_accepting_dicts_iterator() -> None: 
```

Comment:
You have to make sure you run `black` before committing.  You have an extra space at the end of this line.

Reply:
Done Sir